src/alr/alr-build_env.adb

```diff
@@ -0,0 +1,175 @@
+with Ada.Strings.Unbounded;
+with Ada.Text_IO;
+
+with GNAT.OS_Lib;
+
+with Alire.Utils;
+with Alire.Properties.Scenarios;
+with Alire.GPR;
+with Alire.Directories;
+
+with Alr.Platform;
+with Alr.Paths;
+with Alr.OS_Lib;
+with Alr.Query;
+with Alr.Commands;
+
+package body Alr.Build_Env is
```

Comment:
I think this is one of these changes, as you're creating new files, where you can take advantage to move core functionality from `alr` the tool into `alire` the lib. What do you think?

Reply:
I was going to but this in `Alire.` but there are dependencies in `Alr.`. Do you think I should try to move everything to `Alire.`?

Reply:
You're right, this might not be easy. I'd say, don't lose time over it.